runner/runners/local_output.go

```diff
@@ -101,14 +102,23 @@ func (s *localOutputCreator) ServeHTTP(w http.ResponseWriter, r *http.Request) {
     xhr.onreadystatechange = function () {
       var DONE=4, OK=200;
       if (xhr.readyState === DONE && xhr.status == OK) {
+        var id = xhr.getResponseHeader('X-Resource-Id')
+        var txt = xhr.responseText.substring(prevLength);
+        prevLength = xhr.responseText.length
+        if resourceId == ''
+          resourceId = id
+        if id != resourceId {
+          txt = Date() + ': Underlying resource changed! Quitting.'
```

Comment:
recommend give more info in this response and put marker in to highlight it.... 

*************************
Underlying resource changed - quitting streaming.
This is typically caused by the worker restarting.  
The file being streamed was: <path to original file>, use ssh to retrieve/view the file.
**************************